src/app/dashboard/learning-object-management/learning-object-management.component.ts

```diff
@@ -74,15 +74,15 @@ export class LearningObjectManagementComponent implements OnInit, OnDestroy {
     this.subToFilterChange();
     this.subToSearchText();
     this.subToItemsPerPage();
-    this.pagerService.setPage(this.query.page);
+    this.pagerService.setPage(this.query.currPage);
     this.subToPage();
   }
 
   navigateTo(route: string, object: LearningObject) {
     const learningObjectId = object.id;
     this.filterSearchService.resetAll();
     this.activeLink = route;
-    this.router.navigate([`/${route}`, learningObjectId ], { skipLocationChange: true });
+    this.router.navigate([`${route}`, learningObjectId ], { skipLocationChange: true });
```

Comment:
Why use string interpolation here over a string literal?

Reply:
How about this instead:

```suggestion
    this.router.navigate([route, learningObjectId ], { skipLocationChange: true });
```